data/p10/FAPITargetsNameMapList.lsv

```diff
@@ -23,7 +23,7 @@ TARGET_TYPE_OMIC:unit-omic-power10
 TARGET_TYPE_OMI:unit-omi-power10
 TARGET_TYPE_OCMB_CHIP:chip-ocmb,chip-ocmb-planar
 TARGET_TYPE_MEM_PORT:unit-mem_port
-TARGET_TYPE_DIMM:lcard-dimm-ddimm,lcard-dimm-ddimm4u,lcard-dimm-ddr4
+TARGET_TYPE_DIMM:lcard-dimm-ddimm,lcard-dimm-ddimm4u,lcard-dimm-ddr4,unit-ddr,unit-ddr4-jedec,unit-ddr5-jedec
```

Comment:
These are now the 'dimm types' hence I added them here. We did a similar thing for Denali. But we don't have an MRW with DDR5 for Denali. So all that was verified was prevention of regression.

Reply:
matches with hostboot changes

Reply:
I'm not sure what this association is actually doing but it isn't accurate anymore to say that a FAPI DIMM target is represented by the lcard-dimm-ddimm targets in the MRW.

Sidenote - Eventually I hope to transition the MRWs to all use the generic "unit-ddr" but that will take awhile (if it ever happens).

Reply:
> These are now the 'dimm types' hence I added them here. We did a similar thing for Denali. But we don't have an MRW with DDR5 for Denali. So all that was verified was prevention of regression.

If all are applicable for the BMC box means ok. Suppose only a few variants of DIMM only we will support in the BMC box then I would recommend adding only the required targets type instead of all.

> I'm not sure what this association is actually doing

It's used to process the ekb attributes xml file along with MRW file. Some more [details](https://github.com/open-power/pdata/blob/main/scripts/dt_generate.sh#L86)

> but it isn't accurate anymore to say that a FAPI DIMM target is represented by the lcard-dimm-ddimm targets in the MRW.

Oh ok, I don't know about this, is it because of the DDR supports?
I think we need to understand a little bit more about this and handle it properly @aravynd fyi.

> Eventually I hope to transition the MRWs to all use the generic "unit-ddr" but that will take awhile (if it ever happens).

Oh ok thanks for sharing the details, I think we need to keep an eye on this.